clean_docker_images.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
can you move this script under scripts directory so that the structure becomes a little more cleaner?

Reply:
sure

Reply:
it seems some tests on the VM failed but I hardly believe it is due to this little modification...
Once logged in Azure pipelines website, is a bit weird.
The stage tests is in red alhough It is written "100% tests passed".
I clicked on "rerun the test stage" but I cannot see any effect.
Any hint about this ?

Reply:
Let's try a rebase on current master and see.